src/components/Modal/Modal.css

```diff
@@ -0,0 +1,28 @@
+.modal-container {
+    height: 320px;
+    width: 450px;
+    
+}
+
+input[type=text], input[type=password], input[type=email] {
+    width: 100%;
+    padding: 12px 20px;
+    margin: 8px 0;
+    display: inline-block;
+    border: 1px solid #ccc;
+    box-sizing: border-box;
+    border-radius: 20px;
+}
+
+.label-input {
+    margin-top: 50px;
+}
+
+.button {
+    background-color: #333333 !important;
```

Comment:
Don't use importants.

Reply:
easy to say, but Idk what to use instead. Here I have to change color of a button from material-ui